provider-messaging/src/hooks/useWebSocket.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+/* TODO: I'll transform this into hooks if I'll fully grasp the */
```

Comment:
Seems like the comment got cut off here.

Reply:
@dimdejesus For TODOs, you may also want to put your (or someone else's) name on it.